AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -120,6 +122,25 @@ private void initializeDisplayedParticipantsLiveData() {
         callingContext.getDisplayedParticipantsLiveData().observe(this, observerDisplayedRemoteParticipants);
     }
 
+    private void initializeRemoteParticipantUpdateObserver() {
+        final Observer<RemoteParticipantUpdate> observerRemoteParticipantUpdate = remoteParticipantUpdate -> {
+            final String id = callingContext.getId(remoteParticipantUpdate.getRemoteParticipant());
+            final ParticipantView pv;
+            if (participantIdIndexPathMap.containsKey(id)) {
+                final int prevIndex = participantIdIndexPathMap.get(id);
+                pv = participantViewList.get(prevIndex);
+
+                switch (remoteParticipantUpdate.getUpdateType()) {
```

Comment:
remoteParticipantUpdate.getUpdateType() can we directly send updateType here? then we do not need getUpdateType()?